File: backend/app/src/main/java/com/lemon/DTO/TransferResponse.java
package com.lemon.DTO;

import com.lemon.models.TransactionStatus;
import java.math.BigDecimal;
import java.time.Instant;


public record TransferResponse(
        String reference,
        TransactionStatus status,
        BigDecimal amount,
        String toPhoneNumber,
        Instant createdAt
) {}

File: app/src/main/java/com/example/lemon/network/models/TransferResponse.kt
package com.example.lemon.network.models

import com.example.lemon.model.TransactionStatus
import com.example.lemon.model.TransactionType
import kotlinx.serialization.Contextual
import kotlinx.serialization.Serializable
import java.math.BigDecimal

@Serializable
data class TransferResponse(

    val reference: String,
    val status: String,
    val amount: String,
    val toPhoneNumber: String,
    val createdAt: String
)